src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -698,6 +707,14 @@
                         });
                     }
 
+                    /* Prevents from disallowing clicks on "copy to clipboard" elements inside toggles */
+                    var copyToClipboardElements = toggles[i].querySelectorAll('span[data-clipboard-text]')
+                    for (var k = 0; k < copyToClipboardElements.length; k++) {
+                        addEventListener(copyToClipboardElements[k], 'click', function(e) {
+                            e.stopPropagation();
```

Comment:
IMHO we could always stopPropagation (and not ONLY for items in SPAN/toggle). And moving this logic at line 44